src/controllers/User/GenerateOtp.js

```diff
@@ -13,21 +11,21 @@ const schema = Joi.object().keys({
 exports.validator = validator.body(schema);
 
 exports.handler = async (req, res, nex) => {    
- const  userId    = req.body.userId; 
- const  new_otp   = Math.floor(Math.random() * 1000000);
-
-  try {    
-    const userInsertData = User.findByIdAndUpdate(userId,
-        {$push: {otp: new_otp}},
-        {safe: true, upsert: true},
-        function(err, doc) {
-            if(err){
-              res.fail("otp not send");
-            }else{
-              res.success("otp send", new_otp);
-            }
+    const  userId    = req.body.userId; 
+    const  new_otp   = Math.floor(100000 + Math.random() * 900000);
+  try {   
+    const user = await User.findById(userId);
+    if(user.otp.length+1 <=3){
```

Comment:
Remove this condition

Reply:
The condition is not going to be for otp sending it is going to be for otp attempt, also for that create global config MAX_OTP_ATTEMPT and check from there